automations/profilefollower.py

```diff
@@ -57,18 +62,27 @@ def execute(self):
             pos = (left_pos + right_pos) / 2
             pos_error = linear_seg[0] - pos
 
+<<<<<<< HEAD
             linear_output = (self.kP * pos_error + kV * linear_seg[1]
             + kA * linear_seg[2])
             linear_output = (
+=======
+            self.sd.putNumber("distance_error_mp", pos_error)
+
+            linear_output = \
+>>>>>>> 0d9a851... Get motion profiling (linear and angular) working and add in some semi-tuned constants.
```

Comment:
Wait a second. You committed a merge conflict here. This will break `git bisect`, yadda yadda.